make_prg/subcommands/from_msa.py

```diff
@@ -52,95 +75,94 @@ def register_parser(subparsers):
         ),
     )
     subparser_msa.add_argument(
-        "-o",
-        "--outdir",
-        dest="output_dir",
-        action="store",
-        default=".",
-        help="Output directory. Default: %(default)s",
-    )
-    subparser_msa.add_argument(
-        "-n",
-        "--prg_name",
-        dest="prg_name",
-        action="store",
-        help="Prg file name. Default: MSA file name",
-    )
-    subparser_msa.add_argument(
-        "-S",
-        "--seqid",
-        help="Sequence identifier to use for the output sequence/PRG. Default is the file name",
-    )
-    subparser_msa.add_argument(
-        "--no_overwrite",
-        dest="no_overwrite",
+        "--output_graphs",
+        dest="output_graphs",
         action="store_true",
-        help="Do not replace an existing prg file",
+        default=False,
+        help="Outputs the recursive tree graphical representation (for development use only)",
     )
-    subparser_msa.add_argument(
-        "-O",
-        "--output-type",
-        help="p: PRG, b: Binary, g: GFA, a: All. Combinations are allowed i.e., gb: GFA and Binary. Default: %(default)s",
-        default="a",
-        type=OutputType,
-    )
-    subparser_msa.add_argument("--log", help="Path to write log to. Default is stderr")
```

Comment:
I strongly disagree with this. Is there a functional reason why this is no longer possible? We only recently added this option in.

Reply:
There is no functional reason. Will add this back

Reply:
Fixed in https://github.com/iqbal-lab-org/make_prg/pull/35/commits/4e8fdf5f99213aa664e724458bd5de466ee1066d and https://github.com/iqbal-lab-org/make_prg/pull/35/commits/698827c1c6b7200b657587b6af8610ceb71e762f , please check. Also, I've added 6 more integration tests just to ensure we are building the correct output when different `--output-type` parameters are given, but these integration tests are not part of this PR, will be added in a future PR.

Reply:
Great! Cheers.